inventory.schema.json

```
@@ -19,6 +19,13 @@
         "2012-01-01"
       ],
       "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+    },
+    "dateordatetime": {
+      "type": "string",
+      "examples": [
+        "2018-10-11 04:55:07"
+      ],
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}([ |T][0-9]{2}:[0-9]{2}:[0-9]{2}(Z|[+|-]([0-9]{2}):([0-9]{2}):([0-9]{2})))??$"
```

1. Could be simplier.
2. `)??`->`?)?` at the end of pattern.
```suggestion
      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}([ |T][0-9]{2}:[0-9]{2}:[0-9]{2}(Z|[+|-][0-9]{2}:[0-9]{2}:[0-9]{2})?)?$"
```